exchange/xgrid.F90

```diff
@@ -464,8 +460,8 @@ module xgrid_mod
 ! Include variable "version" to be written to log file.
 #include<file_version.h>
 
- real(r8_kind), parameter                              :: EPS = 1.0d-10
- real(r8_kind), parameter                              :: LARGE_NUMBER = 1.d20
+ real(r8_kind), parameter                              :: EPS = 1.0e-10_r8_kind
+ real(r8_kind), parameter                              :: LARGE_NUMBER = 1.e20_r8_kind
```

Comment:
Is this "more correct" than using `d`?

Reply:
I think so, `d` will double whatever the current default real precision is so it could give you 16 byte reals, this will always be 8 bytes.

Reply:
Can you give us a reference to this as it has not been my experience.

Reply:
I had thought that from testing it before, but I just realized that it's actually compiler specific. I just tested it with:
```
print *, sizeof(1.e0), sizeof(1.d0)
```
With default 8 byte reals, GCC prints 8 and 16 and intel prints 8 and 8.

I guess this should still use the `_r8_kind` because that'll always be 8.

Reply:
So if you do this, then it's going to change the precision and change answers?

Reply:
regardless, it will be a 8bit real. With intel theres no change. With gnu it'll be implicitly casted so there shouldn't be a change, at least there wasn't when i tested it.